src/PlugKeyRing/modules/NetworkModule/Network.ts

```diff
@@ -9,6 +9,7 @@ import { DEFAULT_MAINNET_TOKENS, TOKENS } from "../../../constants/tokens";
 import { StandardToken } from "../../../interfaces/token";
 import { recursiveParseBigint } from "../../../utils/object";
 import { createAgent } from "../../../utils/dfx";
+import { SignIdentity } from '@dfinity/agent';
```

Comment:
We usually put the import from packaged at the top to maintain an order

Reply:
We should add aliases to the code honestly